src/generator/default/dbmodel.php

```diff
@@ -47,11 +51,24 @@ public function get<?= $relation->getCamelName() ?>()
     {
         return $this-><?= $relation->getMethod() ?>(\<?= trim($relationNamespace, '\\') ?>\<?= $relation->getClassName() ?>::class, <?php
             echo str_replace(
-                    [',', '=>', ', ]'],
-                    [', ', ' => ', ']'],
-                    preg_replace('~\s+~', '', VarDumper::export($relation->getLink()))
-            )
+    [',', '=>', ', ]'],
+    [', ', ' => ', ']'],
+    preg_replace('~\s+~', '', VarDumper::export($relation->getLink()))
+)
```

Comment:
bug in style fixer or is this change on purpose?

Reply:
style-fixer